README.md

```diff
@@ -1,157 +1,22 @@
-# Cloud Spanner Banking Application
+# Cloud Spanner Sample Apps
 
-This repository provides the source code for a Cloud Spanner sample financial
-application. It consists of 2 components:
-1. A gRPC server that provides application level functionality
-  (see: [service.proto](server/src/main/proto/service.proto)) using CloudSpanner
-  as the storage backend, found in the [server](server) directory
-2. A workload generator for the above, found in the [workload](workload) directory
+This repository provides the source code for Cloud Spanner sample
+applications.
 
-The goal of the sample is to provide a simple working example to jump-start
-exploring Cloud Spanner.
+These applications highlight how to develop applications to work with Cloud Spanner.
 
-## Architecture & Schema
+More details of each sample application can be found in the README for the sample
+app.
 
-### Application Components
-This is how everything fits together:
-![FinAppConnect.png](images/FinAppConnect.png)
+Sample applications:
+- [Banking](finance/README.md)
 
-The FinAppServer is the backend of the application. It provides a gRPC interface
-with application-level operations and uses Cloud Spanner as the storage backend.
-We provide two alternative implementations for connecting to Cloud Spanner for
-educational purposes:
-- [Cloud Spanner Java Client](https://github.com/googleapis/java-spanner)
-- [Cloud Spanner JDBC Client](https://github.com/googleapis/java-spanner-jdbc)
 
-Clients, including the workload generator, a potential frontend component, or
-tools like [grpc_cli](https://github.com/grpc/grpc/blob/master/doc/command_line_tool.md)
-can connect to the FinAppServer using gRPC.
+## Code of Conduct
+Please review the [Code of Conduct](docs/code-of-conduct.md)
 
-### ER Diagram & Operations
+## Contributing
 
-The application models the following entities:
-- *Customer*: End-user of the application. A customer can have one or more
-  accounts.
-- *Account*: A financial account, keeps track of balance. Multiple customers
-  could access a single account with different roles.
-- *Transaction*: A monetary transfer associated with one or more accounts.
+If you are interested in contributing, more details on how to do so are provided 
+in the [Contributing page](docs/contributing.md).
```

Comment:
```suggestion
on the [Contributing page](docs/contributing.md).
```

Nit, feel free to ignore if you disagree.

Reply:
Added in #82